src/main/java/me/arminb/sara/controller/models/QuestionRequest.java

```diff
@@ -0,0 +1,115 @@
+package me.arminb.sara.controller.models;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import me.arminb.sara.entities.Answer;
+import me.arminb.sara.entities.Question;
+
+import java.io.IOException;
+import java.util.Date;
+import java.util.List;
+
+
+public class QuestionRequest {
+
+    public Date getDate() {
+        return date;
+    }
+
+    public void setDate(Date date) {
+        this.date = date;
+    }
+
+    private Date date;
+
+    private String title;
+
+    private String content;
+
+    private List<Answer> answers;
```

Comment:
answers will not be posted with the question request. So I would remove it from this class